src/abi/fuse_abi.rs

```diff
@@ -610,11 +610,17 @@ impl Attr {
             mode: st.st_mode,
             #[cfg(target_os = "macos")]
             mode: st.st_mode as u32,
-            // st.st_nlink is u64 on x86_64 and powerpc64, and u32 on other architectures
-            // ref https://github.com/rust-lang/rust/blob/1.69.0/library/std/src/os/linux/raw.rs#L333
```

Comment:
Please do not remove the useful reference links.

It is better to add another link to the macos implementation (https://github.com/rust-lang/rust/blob/1.69.0/library/std/src/os/macos/raw.rs#L44) so that it is clear why we have these conditional compilations.